samples/issue-comment.yaml

```diff
@@ -0,0 +1,59 @@
+# Prerequisites:
+# - Copy to a new GitHub Actions workflow file in the .github/workflows directory.
+# - Update the environment variables for your setup.
+
+# Overall workflow:
+# 1. If issue comment begins with trigger word, then read feature flag (2nd word).
+# 2. Analyze the experiment.
+# 3. Comment on the issue with the analysis summary.
+
+name: Analyze Experiment (issue comment)
+
+on:
+  issue_comment:
+    types: [created]
+
+# Update for your setup
+env:
+  METRIC_TAGS_ORDER: "Important, Cost, Performance"
+
+jobs:
+  analysis:
+    name: Experiment Analysis
+    if: ${{ startsWith(github.event.comment.body, '/experiment-analysis') }}
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      issues: write
+    steps:
+      - name: Azure login using Federated Credentials
+        uses: azure/login@v2
+        with:
+          client-id: ${{ vars.AZURE_CLIENT_ID }}
+          tenant-id: ${{ vars.AZURE_TENANT_ID }}
+          subscription-id: ${{ vars.AZURE_SUBSCRIPTION_ID }}
+
+      - name: Parse feature flag from issue comment
+        id: parse
+        run: echo "feature-flag=$(echo $COMMENT | tr -s ' ' | cut -d ' ' -f2)" >> "$GITHUB_OUTPUT"
```

Comment:
Unfortunately, I think this is vulnerable to injection attacks. Looks for dev insights on this.

Reply:
Can we check if commenter has sufficient privileges to trigger analysis results ? Example, commenter has permission to trigger github action manually ? If they have permission to trigger github action manually, then we are not permitting any additional thing that they can't already do. 

April , thoughts ?